internal/exec/terraform_outputs.go

```diff
@@ -205,50 +205,50 @@ func execTerraformOutput(
 			return nil, err
 		}
 
-		l.Debug(fmt.Sprintf("Executed 'terraform init %s -s %s'", component, stack))
+		log.Debug(fmt.Sprintf("Executed 'terraform init %s -s %s'", component, stack))
 
 		// Terraform workspace
-		l.Debug(fmt.Sprintf("Executing 'terraform workspace new %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
+		log.Debug(fmt.Sprintf("Executing 'terraform workspace new %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
 		err = tf.WorkspaceNew(ctx, terraformWorkspace)
 		if err != nil {
-			l.Debug(fmt.Sprintf("Workspace exists. Executing 'terraform workspace select %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
+			log.Debug(fmt.Sprintf("Workspace exists. Executing 'terraform workspace select %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
 			err = tf.WorkspaceSelect(ctx, terraformWorkspace)
 			if err != nil {
 				return nil, err
 			}
-			l.Debug(fmt.Sprintf("Executed 'terraform workspace select %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
+			log.Debug(fmt.Sprintf("Executed 'terraform workspace select %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
 		} else {
-			l.Debug(fmt.Sprintf("Executed 'terraform workspace new %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
+			log.Debug(fmt.Sprintf("Executed 'terraform workspace new %s' for component '%s' in stack '%s'", terraformWorkspace, component, stack))
 		}
 
 		// Terraform output
-		l.Debug(fmt.Sprintf("Executing 'terraform output %s -s %s'", component, stack))
+		log.Debug(fmt.Sprintf("Executing 'terraform output %s -s %s'", component, stack))
 		outputMeta, err := tf.Output(ctx)
 		if err != nil {
 			return nil, err
 		}
-		l.Debug(fmt.Sprintf("Executed 'terraform output %s -s %s'", component, stack))
+		log.Debug(fmt.Sprintf("Executed 'terraform output %s -s %s'", component, stack))
```

Comment:
We're mixing atmos terraform syntax and terraform syntax which I find confusing. Nonethless, reformatting what was here... 
```suggestion
		log.Debug("Executed terraform output command",
    "command", fmt.Sprintf("terraform output %s -s %s", component, stack),
    "component", component,
    "stack", stack,
)
```